source/MaterialXRuntime/RtFileIo.h

```diff
@@ -46,6 +48,19 @@ class RtCoreIo : public RtApiBase
     /// If a filter is used only elements accepted by the filter
     /// will be written to the document.
     void write(DocumentPtr& doc, WriteFilter filter = nullptr);
```

Comment:
We could hide the DocumentPtr read/write methods now that we renamed the class and operate on files instead.

Reply:
There is still a test case which performs a document comparison which I haven't transposed yet. Need to figure out how to do that first.